Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2999,6 +3000,15 @@ extension BrowserViewController: IntroViewControllerDelegate {
             }
         }
     }
+    
+    func introViewControllerShouldAskForNotifications(_ introViewController: IntroViewController) -> Bool {
+        // Only include the slide if Leanplum has decided it needs to run and if the current locale is en_US/GB/CA/AU/HK/SG
+        return Leanplum.hasStarted() && Locale.current.identifier.startsWith("en_")
```

Comment:
A check against the enabled locales makes sense. Maybe expose https://github.com/mozilla-mobile/firefox-ios/blob/master/Client/Application/LeanplumIntegration.swift#L90 in a public method and then use it both internally and externally?